.gitignore

```diff
@@ -1,5 +1,8 @@
 .DS_Store
 node_modules/
+bin/*
+!bin/install-php-linter
+.vscode
```

Comment:
The .gitignore file that goes into the projects should be used only to ignore files related to the project itself. Things like editor configuration, which are specific to your own local dev environment, should go into your [local global ignore file](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)